app/components/post_list/post/header/header.tsx

```diff
@@ -126,6 +131,13 @@ const Header = (props: HeaderProps) => {
                         style={style.time}
                         testID='post_header.date_time'
                     />
+                    {isPostPriorityEnabled && post.props?.priority && (
```

Comment:
if `priority` is an empty string, this will cause problems, do wrap it a Boolean.

`Boolean(isPostPriorityEnabled && post.props?.priority)`